src/pages/profile/index.js

```diff
@@ -1,81 +1,108 @@
 import React, { useState, useEffect, useContext } from "react";
 import axios from "axios";
-import { GoBackButton } from '../../components/BackLink';
-import './profile.scss';
-import userImageProfile from './images/user-image-profile.png';
-import linkArrow from './images/linkArrow.png'
-import { motion } from 'framer-motion';
-import { ProfileConfigOptions } from '../../components/profileConf/profileConfig/index.js';
+import { GoBackButton } from "../../components/BackLink";
+import "./profile.scss";
+import userImageProfile from "./images/user-image-profile.png";
+import linkArrow from "./images/linkArrow.png";
+import { motion } from "framer-motion";
+import { ProfileConfigOptions } from "../../components/profileConf/profileConfig/index.js";
 import { UserSessionValidation } from "../../components/UserSessionValidation";
 import { UserContext } from "../../context/UserContext";
 
 function Profile() {
   const { login, userSession } = useContext(UserContext);
-
   const [settingsOpen, setSettingsOpen] = useState(false);
   const [chUserName, setChUserName] = useState(false);
   const [chPassword, setChPassword] = useState(false);
-  const linkAnimateFrom = {opacity: 0, y: -40};
-  const linkAnimateTo = {opacity: 1, y: 0};
-  return (
+  const linkAnimateFrom = { opacity: 0, y: -40 };
+  const linkAnimateTo = { opacity: 1, y: 0 };
+  if (!userSession) {
+    return <UserSessionValidation />;
+  } else {
+
+    return (
     <>
-    <UserSessionValidation/>
-    <div className="profilePageContainer">
-      <header className="profile-header"> <GoBackButton/> </header>
-      <main className="profile-main">
-        <div className="user">
-          <picture>
-            <img alt="user photo" data-testid="user--photo" src={userImageProfile}></img>
-          </picture>
-          <h1>{userSession.nickname}</h1>
-        </div>
-        <section className="options" data-testid="options">
-          <motion.div
-            initial={linkAnimateFrom}
-            animate={linkAnimateTo}
-            transition={{delay: 0.10}}
-          >
-            <button type="button" onClick={()=> setSettingsOpen(!settingsOpen)}> Settings <img src={linkArrow}></img></button>
-          </motion.div>
-            { settingsOpen &&
+      <div className="profilePageContainer">
+        <header className="profile-header">
+          {" "}
```

Comment:
what is this? :v

Reply:
I forgot to delete the brackets of the render conditional, but I already deleted them. thank you!